lib/utilityservice.js

```diff
@@ -764,22 +764,28 @@ UtilityService.prototype.getDataPackKeyFromFilename = function(filename, jobInfo
     return null;
 }
 
-UtilityService.prototype.setVlocitySetting = async function(settingName, value) {
+UtilityService.prototype.setVlocitySetting = async function(settingName, value, jobInfo) {
+    var customSettingAPIName = `${this.vlocity.namespacePrefix}GeneralSettings__c`;
+    var valueColumn = `${this.vlocity.namespacePrefix}Value__c`;
+    if (jobInfo.generalSettingsCustomSetting) {
+        customSettingAPIName = jobInfo.generalSettingsCustomSetting
+        valueColumn = 'Value__c';
```

Comment:
What is this value column? Is this meant to be for OmniStudio OmniInteractionConfig? It should be Value if so. And I am not sure Set will work for this without using the tooling API since it moved to Metadata.

Reply:
valueColumn is the API name of the Customsetting Colum that has the Actual value of the setting. The idea is that The customer creates a new CustomSetting with a column named 'Value__c'. The new Custom Setting API Name will be added by the customer tp the jobfile using "generalSettingsCustomSetting:<xxx>"